Adding Auras, Statuses, Unfoldings and Followers in *.json

Barondil opened the pull request:
Adding Auras, Statuses, Unfoldings and Followers for DE and EN.json.

Barondil committed:
Update de_DE.json

Adding auras, outcome, unfolding & followers

Barondil committed:
Update de_DE.json

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update en_US.json

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update de_DE.json

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update de_DE.json

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update en_US.json

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update en_US.json

Fix: "Campaign": and add ,

Adding Auras, Statuses, Unfoldings and Followers

Barondil commented:
There is according to https://jsonlint.com/ still a parse error.
But I cannot find exactly, where the error is. 
Has someone an idea? 

![grafik](https://github.com/marcojanssen/drunagor-helper/assets/163357376/0e5e0a8c-e1dc-4823-a485-31963ba830a5)

erikblessman commented:
The end of the violating line is not shown, but the error suggests that
there is a closing curly brace "}" at the end of that line. That would be
an extra curly brace that should be removed. (Note: the closing curly brace
on the following line needs to stay.)
On Tue, Apr 30, 2024, 14:32 Barondil ***@***.***> wrote:

> There is according to https://jsonlint.com/ still a parse error.
> But I cannot find exactly, where the error is.
> Has someone an idea?
>
> grafik.png (view on web)
> <https://github.com/marcojanssen/drunagor-helper/assets/163357376/0e5e0a8c-e1dc-4823-a485-31963ba830a5>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marcojanssen/drunagor-helper/pull/72#issuecomment-2087144202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQGQ6BA2GK3C7LMAVVUPJODY775V3AVCNFSM6AAAAABG4B4RVSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAOBXGE2DIMRQGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

Barondil commented:
Thank you for your tip. And sorry that I didn't post the complete code. 
I think I found my mistake. In the violating line, there is a "," at the end.
Not only in this, but in every single line with _"effect":_  xD

Barondil committed:
Update en_US.json

#2 Fix: Delete commas

#1 Fix: "Campaign": and add ,

Adding Auras, Statuses, Unfoldings and Followers

Barondil committed:
Update de_DE.json

#2 Fix: Delete commas

#1 Fix: "Campaign": and add ,

Adding Auras, Statuses, Unfoldings and Followers
